webcompat/static/js/lib/bugform.js

```diff
@@ -467,6 +470,18 @@ function BugForm() {
     }, this));
   };
 
+  this.autoExpand = function() {
+    $(document).one("focus.js-autoExpand", "textarea.js-autoExpand", function() {
+      this.baseScrollHeight = this.scrollHeight;
+    })
+    .on("input.js-autoExpand", "textarea.js-autoExpand", function() {
+      var minRows = 2;
```

Comment:
minRows is the number of rows that I would want to add in the total number of rows.